community-addon-template/tests/setup/suite.ts

```diff
@@ -21,24 +21,50 @@ const SETUP_DIR = fileURLToPath(new URL('.', import.meta.url));
 
 type Fixtures<Addons extends AddonMap> = {
 	page: Page;
-	run(variant: ProjectVariant, options: OptionMap<Addons>): Promise<string>;
+	run(addonTestCase: Flavor<Addons>): string;
```

Comment:
I think you actually forgot to rename the types in your last commit, shouldn't this be `AddonTestCase<>` now?

Reply:
🫣 oupy!

_I don't really know why, but with this branch I have a mix with remotes_